homeassistant/components/tts/__init__.py

```diff
@@ -316,6 +317,10 @@ def async_register_engine(self, engine, provider, config):
             provider.name = engine
         self.providers[engine] = provider
 
+        platform_path = PLATFORM_FORMAT.format(domain=engine, platform=DOMAIN)
+        if platform_path not in self.hass.config.components:
+            self.hass.config.components.add(platform_path)
```

Comment:
It's a set, so no need to check
```suggestion
        self.hass.config.components.add(PLATFORM_FORMAT.format(domain=engine, platform=DOMAIN))
```

Reply:
Perfect 👍